h3-quinn/src/lib.rs

```diff
@@ -362,6 +383,17 @@ impl RecvStream {
             read_chunk_fut: ReusableBoxFuture::new(async { unreachable!() }),
         }
     }
+
+	/// Check if this stream has been opened during 0-RTT.
+	///
+	/// In which case any non-idempotent request should be considered dangerous at the application
+	/// level. Because read data is subject to replay attacks.
+    pub fn is_0rtt(&self) -> bool {
+        self.stream
+            .as_ref()
+            .map(|s| s.is_0rtt())
+            .unwrap_or(false)
```

Comment:
when `self.stream` is `None` this will always return `false`.
IIRC this can only happen if the user uses the `poll` API of the streams wrong. If so this function would return `false` even for `0rtt` request. 

We could just change this to `expect`.
The better way IMO would be to call `is_0rtt` in the `new` function and store the result in a new field for this function to return.